rust_cli/src/codama_sdk/programs.rs

```diff
@@ -9,5 +9,5 @@ use solana_program::{pubkey, pubkey::Pubkey};
 
 
   /// `upload_idl_anchor` program ID.
-  pub const UPLOAD_IDL_ANCHOR_ID: Pubkey = pubkey!("idLB41CuMPpWZmQGGxpsxbyGDWWzono4JnFLJxQakrE");
+  pub const UPLOAD_IDL_ANCHOR_ID: Pubkey = pubkey!("F1kv57eE7ikexSKNboUEeSCJ8WKuE1ZxfDqfv15UbraX");
```

Comment:
Was this an intended change? In production it should use idLB41CuMPpWZmQGGxpsxbyGDWWzono4JnFLJxQakrE right?

Reply:
Will revert this to original program id

Reply:
In the current state on the branch the idLB... program id is in use